src/components/profile/ProfileDirective.js

```diff
@@ -15,6 +15,7 @@ goog.require('ga_profile_service');
           templateUrl: 'components/profile/partials/profile.html',
           scope: {
             feature: '=gaProfile',
+            map: '=gaProfileMap',
```

Comment:
BTW: this might probably be a good opportunity to add a new feature as well. Right now we related mouse-move from profile to map (by drawing the overlay on the map here). We could also draw the point on the profile when element is selected in map (to point out in profile where we are). Both could probably be done in a map-aware service, with some loose, message based coupling

Thoughts?